ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py

```diff
@@ -1132,3 +1132,35 @@ def test_jax_numpy_arcsinh(
         fn_tree="numpy.arcsinh",
         x=x[0],
     )
+    
+    
+# argmin
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("float")),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.jax.numpy.argmin"
+    ),
+    keepdims=st.booleans(),
```

Comment:
You are missing the axis argument. You can use `dtype_values_axis` from `helpers` instead of `dtype_and_values`.